compose.yaml

```diff
@@ -0,0 +1,37 @@
+services:
```

Comment:
Is the compose.yaml file used later? As far as I can tell, it looks like the docker-compose-base.yaml / docker-compose-dev.yaml should suffice but maybe I missed it.

Reply:
The `compose.yaml` file is used in all Rust service in the VSCode Devcontainer configuration, it is similar to `docker-compose-dev.yaml`, except for that it does not start the application itself (makes debugging easier)